src/bun.js/bindings/JSBuffer.cpp

```diff
@@ -1094,12 +1094,16 @@ static inline JSC::EncodedJSValue jsBufferPrototypeFunction_toStringBody(JSC::JS
         break;
     }
 
-    case WebCore::BufferEncodingType::latin1:
     case WebCore::BufferEncodingType::ascii: {
         ret = Bun__encoding__toStringASCII(castedThis->typedVector() + offset, length, lexicalGlobalObject);
         break;
     }
 
+    case WebCore::BufferEncodingType::latin1: {
+        ret = Bun__encoding__toStringLatin1(castedThis->typedVector() + offset, length, lexicalGlobalObject);
```

Comment:
Since all this does is duplicate the string without checking, it will be faster if we do it like this:

```suggestion
        ret = JSC::jsString(vm, WTF::StringImpl::create(reinterpret_cast<const LChar*>(castedThis->typedVector() + offset), length));
```

This also means the memory will be owned by JSC instead of Bun. In general, I think it's best for JSC to own as much memory as possible because JSC's codebase is much more tested in production than Bun is right now.